packages/dr.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class dr < Package
+  description 'A simple ed-like devRant client'
+  homepage 'https://github.com/Ewpratten/dr'
+  version '1.0-cros'
+  source_url 'https://codeload.github.com/Ewpratten/dr/tar.gz/1.0-chr'
+  source_sha256 '3e7b33bec353b23132db365e5bacd175361d1e9039544e5e64b8df8c7e828006'
+
+  depends_on 'python3'
+
+  def self.build
+    system "cd build/"
+    system 'sh ./build-cros.sh'
+  end
+
+  def self.install
+    system "which dr"
```

Comment:
This step will require a bit more work.  You will need to add all the files created in the build step above to `#{CREW_DEST_PREFIX}` so the metadata that tracks the files can be generated.  This process is basically needed for operations such as `crew files dr` and `crew uninstall dr`.

Reply:
the idea is that the only file created is the dr file in /usr/local/bin

I want everything else to be deleted. How do I do that?

Reply:
You will have to compile all the python3 dependencies/modules in with the dr executable.  From what I gather, that still needs to be done yet?